code/modules/Economy/Wage-Process.dm

```diff
@@ -0,0 +1,8 @@
+var/global/datum/controller/process/wagePayout/wagePayoutController = null
+/datum/controller/process/wagePayout
+	schedule_interval = 9000 // 15 minutes
```

Comment:
You mean like how we can say sleep(15 SECONDS)?
I'm not sure how that'd be done honestly